src/configure

```diff
@@ -1518,7 +1518,7 @@ Optional Features:
                           (default is NO).
   --enable-cuda-streams   Use CUDA streams (default is YES).
   --enable-cusparse       Use cuSPARSE (default is YES).
-  --enable-cusolver       Use cuSOLVER (default is NO).
+  --enable-cusolver       Use cusolver (default is NO).
```

Comment:
Should the comment say "Use cuSOLVER"?  This would change `configure.in` as well.  The `configure` on `master` uses capital letters, but presumably `configure.in` does not?  I'm still confused about the inconsistencies here, but I guess this will clean everything up.

Reply:
Fixed. Thanks @rfalgout !

About the inconsistencies, I meant whenever we run the script `./config/update.sh`, the file `configure` should not be changed if it was generated by the script before. This was not true when I ran it with the latest master branch. I don't know how it could happen. I hope this gives what we want. Thanks again!